src/shared/lib/notification/notification.ts

```diff
@@ -0,0 +1,19 @@
+import { Messages } from 'entities/Task';
+
+export const getPermission = () => {
+    if ('Notification' in window) {
+        if (Notification.permission === 'granted') {
+            return;
+        }
+        if (Notification.permission === 'default' || Notification.permission === 'denied') {
```

Comment:
seems it can be just `Notification.permission !== 'granted'`

Reply:
Fixed